azure-devops/ioweb/07_io-web-profile.tf

```diff
@@ -43,9 +43,26 @@ locals {
     TF_PUBLIC_JWT_SPID_LEVEL_VALUE_L1 = "https://www.spid.gov.it/SpidL1"
     TF_PUBLIC_JWT_SPID_LEVEL_VALUE_L2 = "https://www.spid.gov.it/SpidL2"
     TF_PUBLIC_JWT_SPID_LEVEL_VALUE_L3 = "https://www.spid.gov.it/SpidL3"
-    git_email                         = module.secrets_azdo.values["azure-devops-github-EMAIL"].value
-    git_username                      = module.secrets_azdo.values["azure-devops-github-USERNAME"].value
-    github_connection                 = azuredevops_serviceendpoint_github.azure-devops-github-rw.service_endpoint_name
+    TF_PUBLIC_API_BASE_URL            = "https://api-web.io-pagopa.it/ioweb-backend"
+    TF_PUBLIC_API_BASE_PATH           = "/api/v1"
+    TF_PUBLIC_API_FETCH_TIMEOUT       = 5000
+    TF_PUBLIC_API_FETCH_MAX_RETRY     = 5
+    TF_PUBLIC_FEATURE_FLAG            = false
+    TF_PUBLIC_SPID_CIE_ENTITY_ID      = "xx_servizicie"
```

Comment:
```suggestion
    TF_PUBLIC_CIE_ENTITY_ID      = "xx_servizicie"
```

Reply:
fixed in https://github.com/pagopa/io-azure-devops/pull/25/commits/27dfd1ae310f4ef1e35487f51b41a4e2c8fc9e3b